### ISSUE TITLE
Installed library missing executable permissions

### ISSUE THREAD

**Issue by adamjstewart** [flame/blis#343]
```console
$ ldd -r libblis.so
ldd: warning: you do not have execution permission for `libblis.so'
	linux-vdso.so.1 =>  (0x00007ffe15d78000)
	libm.so.6 => /lib64/libm.so.6 (0x00002ac685d2c000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00002ac68602e000)
	librt.so.1 => /lib64/librt.so.1 (0x00002ac68624a000)
	libc.so.6 => /lib64/libc.so.6 (0x00002ac686452000)
	/lib64/ld-linux-x86-64.so.2 (0x00002ac68562c000)
$ ls -l
total 7.3M
-rw-r--r-- 1 adamjs5 19fa-astr510-eng 4.3M Sep 23 15:14 libblis.a
lrwxrwxrwx 1 adamjs5 19fa-astr510-eng   16 Sep 23 15:14 libblis.so -> libblis.so.2.0.0
lrwxrwxrwx 1 adamjs5 19fa-astr510-eng   16 Sep 23 15:14 libblis.so.2 -> libblis.so.2.0.0
-rw-r--r-- 1 adamjs5 19fa-astr510-eng 3.1M Sep 23 15:14 libblis.so.2.0.0
```
This is with BLIS 0.6.0 on CentOS 7 built with GCC 7.2.0.

@fgvanzee

**Comment by fgvanzee** [comment]
Thanks Adam. This should be fixed now in bc16ec7.

**Comment by adamjstewart** [comment]
I can confirm that your patch fixes the problem for me, thanks!
